frontend/src/components/shop/ProductDisplay.tsx

```diff
@@ -0,0 +1,52 @@
+import { Badge, Box, Center, Flex, Image, LinkBox, LinkOverlay, Spacer, VStack } from "@chakra-ui/react"
+import React, { FC } from "react"
+
+const ProductDisplay: FC<{
+    name: string
+    image: string
+    brandName: string
+    price: string
+    link: string
+}> = ({ name, image, brandName, price, link }) => {
+    return (
+        <div>
+            <Box pt-6 background="white" width="223px" height="261px" borderWidth="1px" borderRadius="lg" overflow="hidden">
```

Comment:
What is width 233px and height 261px, is this going the be responsive on mobile right ?

Reply:
Yes, we will make it responsive in next Sprint

Reply:
The width and height for now are only for making the component initially

Reply:
Ok then there is no problem here.